docs/examples/kwok-in-local.md

```diff
@@ -0,0 +1,35 @@
+# Run Kwok in the Local
+
+Set Kwok repository
```

Comment:
could you follow the documentation style commented above in `kwok-in-cluster.md`?